english/challenge-marshmellow-to-rock-off/light-route/joker.md

```diff
@@ -14,4 +14,4 @@ and there is a tiny camera looking at you from above.
 
 Who is on the other side, you cannot tell; but, for now, what will you do?
 
-Go up or go down?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.